pkg/plugins/runtime/k8s/webhooks/injector/injector.go

```diff
@@ -213,6 +226,62 @@ func (i *KumaInjector) meshFor(pod *kube_core.Pod, ns *kube_core.Namespace) (*co
 	return meshResource, nil
 }
 
+// loadCustomPatches loads the ContainerPatch CRDs associated with the given pod, divides out
+// the sidecar and init patches, and concatinates each type into its own list for return.
+func (i *KumaInjector) loadCustomPatches(pod *kube_core.Pod, ns *kube_core.Namespace) (sidecarPatches []mesh_k8s.JsonPatchBlock, initPatches []mesh_k8s.JsonPatchBlock, err error) {
+	patchNames := i.cfg.ContainerPatches
+	if val, exist := metadata.Annotations(pod.Annotations).GetString(metadata.KumaContainerPatches); exist {
+		patchNames = val
+	}
+
+	if patchNames == "" {
+		// Avoid Split returning slice containing empty string
+		return []mesh_k8s.JsonPatchBlock{}, []mesh_k8s.JsonPatchBlock{}, nil
+	}
+
+	for _, patchName := range strings.Split(patchNames, ",") {
+		containerPatch := &mesh_k8s.ContainerPatch{}
+		log.Info("loading patch", "namespace", ns.GetName(), "containerpatch", patchName)
```

Comment:
I don't find this log useful. I'd rather log which patches are applied on specific container